model/project_ref.go

```diff
@@ -2870,7 +2870,22 @@ func GetMessageForPatch(patchID string) (string, error) {
 // on the project admin page.
 func ValidateContainers(ecsConf evergreen.ECSConfig, pRef *ProjectRef, containers []Container) error {
 	catcher := grip.NewSimpleCatcher()
+
+	projVars, err := FindMergedProjectVars(pRef.Id)
+	if err != nil {
+		return errors.Wrapf(err, "getting project vars for project '%s'", pRef.Id)
+	}
+	var expansions *util.Expansions
+	if projVars != nil {
+		expansions = util.NewExpansions(projVars.Vars)
+	}
+
 	for _, container := range containers {
+		image := container.Image
+		if expansions != nil {
+			image, err = expansions.ExpandString(container.Image)
+			catcher.Add(errors.Wrap(err, "expanding container image"))
```

Comment:
nit: this can be simplified down to `catcher.Wrap(err, "expanding container image")`